settings.php

```diff
@@ -56,5 +56,23 @@
 
 $ADMIN->add('modcmsfolder', $settings);
 
+$settings = new admin_settingpage('mod_cms_search_settings', new lang_string('search:settings', 'mod_cms'));
+if ($ADMIN->fulltree) {
+    $sql = "SELECT mcf.id, mcf.name
+              FROM {customfield_category} mcc
+              JOIN {customfield_field} mcf ON mcf.categoryid = mcc.id
+             WHERE mcc.component = 'mod_cms' AND mcc.area = 'cmsfield' AND mcf.type IN ('text', 'textarea')";
+    $areas = $DB->get_records_sql_menu($sql);
+
+    $settings->add(new admin_setting_configmulticheckbox(
+        'mod_cms/search_area',
+        get_string('search:settings:area', 'mod_cms'),
+        get_string('search:settings:area_desc', 'mod_cms'),
+        null,
+        $areas,
```

Comment:
php 7.1 doesn't support this extra comma in the end

Reply:
The comma has been removed.

Reply:
Actually, the setting has been removed.